src/waterfall-chart/utils.ts

```diff
@@ -69,7 +68,6 @@ export function getIntervalAndYPoints(
 }
 
 export function checkIfScaleSufficient(scale: number, maxLabelsCount: number, valueRange: number): boolean {
-  if (maxLabelsCount === 0) return true; // to stop the while loop from checking for sufficient scale with zero maxLabelsCount
```

Comment:
Removed the line since it constitutes dead code; the case where maxLabelsCount = 0 is already addressed earlier in the code.